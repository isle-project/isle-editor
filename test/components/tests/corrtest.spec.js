// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import CorrTest from '@isle-project/components/tests/corrtest';
import heartdisease from '../fixtures/heartdisease.json';


// TESTS //

describe( '<CorrTest />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <CorrTest
			data={heartdisease}
			var1="Cost"
			var2="Age"
		/> );
		expect( output ).toMatchSnapshot();
	});
});
