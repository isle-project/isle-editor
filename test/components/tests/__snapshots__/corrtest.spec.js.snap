// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CorrTest /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <label>
    correlation-test
    :
  </label>
  <div />
  <pre>
    t-test for Pearson correlation coefficient

Alternative hypothesis: True correlation coefficient is not equal to 0

    pValue: 0.0994
    statistic: -1.6499
    95% confidence interval: [-0.1281,0.0111]


  </pre>
</div>
`;
